### ISSUE TITLE
Tutorial contest?

### ISSUE THREAD

**Issue by Geokureli** [HaxeFlixel/haxeflixel.com#185]
@gamedevsam 
It still seems like we're having trouble on-boarding newcomers, I remember talking to Tom Fulp while he were doing the tutorial for the NG HF jam and there was some friction. Demos go a long way for people who learn like me, but I think getting new tutorials would help a ton of people. What if we had a tutorial contest with prizes from the HF funds? I've never heard of this before so it might be a bad idea, we may need a way to prevent people from submitting tutorials for the same game-type. But based on the outcome of the last HF jam, I wouldn't want to do another until we have some new sources for newcomers. I'm imagining .md with full code and interactive previews embedded, just like the RPGTutorial

Alternatively we could just start offering up bounties for certain tutorials

**Comment by NulllStack** [comment]
mark, w...wait can I mark?

**Comment by gamedevsam** [comment]
@Gama11 FYI.

**Comment by Geokureli** [comment]
Right now https://haxeflixel.com/ has one tutorial, and treats it as "THE haxeflixel tutorial". for instance, [the docs page](https://haxeflixel.com/documentation/) lists every step of the tutorial, I fear this wont be scalable as more tutorials are made. We should change the website to highlight various tutorials, and have an easy way to add new ones.

I'm not well versed in web dev so I don't know how big a task that is. Should we allow submissions as external links? That way they can just make them in blogspot or whatever editor they are comfortable with and we can find a simple way to list links on the haxeflixel website

**Comment by gamedevsam** [comment]
@Gama11 @impaler do you guys have any insights / suggestions on how we could scale the number of tutorials on the site?

**Comment by Geokureli** [comment]
another important detail is that this should happen after the [5.0.0 release](https://github.com/HaxeFlixel/flixel/issues/2538). I'm also using this contest to encourage people to upgrade

**Comment by RichardBray** [comment]
I think this is a good idea, although I'm a bit worried having multiple tutorials don't the site might confuse newcomers, as in, they won't know which one to use first.

Would it make sense to have one main tutorial, and then have links to other tutorials as well? A bit like Phaser's page
https://phaser.io/learn

**Comment by gamedevsam** [comment]
Yeah I like that, it would also be less effort to change. I think a full scale re-write of the site is out of scope / too much effort at this time. But it shouldn't terribly difficult to add a list of additional tutorials at the bottom of the page.

**Comment by Geokureli** [comment]
yeah, there is a lot of benefit to having a "newbies start here" tutorial and if keeping a main tutorial makes the site edits easier, then let's keep it, as long as it's possible to add more secondary tutorials in some way.

**Comment by ninjamuffin99** [comment]
to bump this, I think it's a very good idea for a tutorial contest.
the youtube channel [3blue1brown had a "math explanation" contest](https://youtu.be/hZuYICAEN9Y) that I feel could work nicely in a similar format. [(results / follow up video)](https://www.youtube.com/watch?v=cDofhN-RJqg)

i was chattin wit will blanton (@01010111) and he suggested it would be cool to have a sorta "shared art set" of things like tiles, sprites, etc. etc. 

that's just a sidenote though, I think as 5.0.0 getting closer might be nice to think about tutorial contest
